agent/command/github_generate_token.go

```diff
@@ -95,11 +114,17 @@ func (r *githubGenerateToken) Execute(ctx context.Context, comm client.Communica
 	// We write or overwrite the expansion with the new token.
 	conf.NewExpansions.PutAndRedact(r.ExpansionName, token)
 
-	conf.AddCommandCleanup(r.FullDisplayName(), func(ctx context.Context) error {
+	conf.AddCommandCleanup(r.Name(), func(ctx context.Context) error {
```

Comment:
I switched it to name because that's what we use for existing spans. The full display name has the step which I think is not important in this case and it's nicer to keep that consistency